libbeat/processors/add_host_metadata/add_host_metadata.go

```diff
@@ -71,10 +72,54 @@ func (p *addHostMetadata) loadData() {
 		if p.info.OS.Build != "" {
 			p.data.Put("host.os.build", p.info.OS.Build)
 		}
+
+		// IP-address and MAC-address
+		var ipList, hwList = p.getNetInfo()
```

Comment:
I'm thinking to make this information optional, meaning it is not sent by default but we have a config option in the processor to enable it. Not sure yet how we should call the config options, any suggestions?

Reply:
Perhaps
netinfo: enabled/disabled 
or 
add_netinfo: true/false?

Reply:
Looking at the code I was even wondering if we should separate mac address and ip. I think lots of people want to have the list of ip addresses but are less interested in the actual mac address. This could lead to a config like:

```
add_fields: [`ip`, `mac`] 
```
I don't like add_fields name to much but the gist is that we could add more fields in the future here.

@andrewkroh Thoughts?

Reply:
Should we do that for all fields or just for ip and mac?

Reply:
The way I was thinking that the fields we have now are always there. And we only have that for additional fields like the once proposed here.

Reply:
Ok, I'll fix that.

Reply:
> The way I was thinking that the fields we have now are always there. And we only have that for additional fields like the once proposed here.

Personally I'd like be able to fully select each field that is included. Keeping the current set as the default. The behavior would be similar to what you get with Metricbeat `metricsets` (there's a default set, and when you define any value you override the default set).

Reply:
@andrewkroh If I understand this correctly, my config example above would mean `host.name` for example would not be sent?

Reply:
> add_fields: [`ip`, `mac`] 

Correct. That would give you only those two fields. But I wouldn't call it `add_fields` if we went with this behavior.

Reply:
I have time to work on this on friday, so it would be great to have a decision by then.

Reply:
@hypp We had a quick internal chat and decided to go with the proposal for `netinfo.enabled` for now and we can still improve it later.